python/jupyter/src/widget.ts

```diff
@@ -83,7 +83,7 @@ export class ChemiscopeView extends ChemiscopeBaseView {
 
     public render(): void {
         PlausibleTracker.trackEvent('jupyter-show', undefined, {
```

Comment:
We might want to use [`trackPageView`](https://plausible-tracker.netlify.app/globals#trackpageview) instead of `trackEvent`

Reply:
uhm. I just modified what you had there - IDK what actually changes, the result is good enough.
you mean that given that we changed to a fake subdomain now we can ignore the "conversion target" thing and directly record pages?